### PR TITLE
Fix compare-optimizer.sh script

### LINKED ISSUE #39: Add a way to run compare-optimizer with a specific binary

### PR DESCRIPTION
**Pull request by rok-cesnovar**
Fixes #39

### CODE REVIEW

File: compare-optimizer.sh [hunk 1]
```diff
@@ -13,9 +13,13 @@ fi
...
 set -e -x
 
-# run develop version of cmdstan with the nightly stanc3 binary
-cd cmdstan; make clean-all; 
-make -j4 build; make -j4 examples/bernoulli/bernoulli; ./bin/stanc --version; cd ..
+# run with the default optimization level
+cd cmdstan; make clean-all; make -j4 build;
+if [ -n "$3" ] ; then
+    rm cmdstan/bin/stanc
+    cp "$3" cmdstan/bin/stanc
```
Comment (WardBrian) [inline compare-optimizer.sh]: I believe the current directory when this is run is `cmdstan` already, no?

Comment (rok-cesnovar) [reply compare-optimizer.sh]: Indeed, fixed.

**Review by WardBrian — APPROVED** [review]

**Comment by WardBrian** [comment]
Thanks @rok-cesnovar - we will also need to update the stanc3 jenkinsfile to pass the extra param, right?

**Comment by rok-cesnovar** [comment]
Yep, will do that now.

**Pull request merged**